### PR TITLE
Do not include campaign_landing_pages on prod

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/20632

This PR removes CLPs from prod.

## Testing done
N/A

## Screenshots
N/A

## Acceptance criteria
- [x] This PR removes CLPs from prod.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by ncksllvn — APPROVED** [review]
Well done man, you figured this out fast!

File: src/site/stages/build/plugins/create-environment-filter.js [hunk 1]
```diff
@@ -8,6 +8,14 @@ function createEnvironmentFilter(options) {
...
       const file = files[fileName];
 
+      // Do not include CLPs on production.
+      if (
+        environmentName === ENVIRONMENTS.VAGOVPROD &&
+        file.entityBundle === 'campaign_landing_page'
+      ) {
+        delete files[fileName];
+      }
+
```
Comment (ncksllvn) [inline src/site/stages/build/plugins/create-environment-filter.js:18]: i _think_ if this executed by the preview server then `options.isPreviewServer` will be `true`

Comment (kelsonic) [reply src/site/stages/build/plugins/create-environment-filter.js:19]: ah perfect, thanks!

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**